### ISSUE TITLE
compute xmlsec in native go code

### ISSUE THREAD

**Issue by philhug** [RobotsAndPencils/go-saml#2]
currently the xmlsec1 binary is invoked for every signature create and verification.
for production code this should be replaced by a native go library or a by using the C library.

**Comment by mbrevoort** [comment]
I agree a native go implementation of XML signing would be ideal.  I would love if someone created such an implementation ;)

**Comment by fatlotus** [comment]
What about https://github.com/ma314smith/signedxml — would that work for this use-case?

**Comment by heerakli** [comment]
I found this one, but didn't try yet https://github.com/crewjam/go-xmlsec
